### ISSUE TITLE
[4.0] Image Preview For Media Field

### ISSUE THREAD

**Issue by joomdonation** [joomla/joomla-cms#34810]
### Steps to reproduce the issue
1. Create a new banner
2. Select an image for it
3. Save it
4. Edit the banner again and look at the Image

### Expected result
Image previous is being displayed properly

### Actual result
At first, it is displaying the text **No Image Select** (as if the path to the image is wrong) and then about less than one second later, it is being displayed properly

### System information (as much as possible)
Joomla 4 nightly build

### Additional comments
Maybe when the page is loaded, the wrong preview image is used and the right path is updated later?

**Comment by dgrammatiko** [comment]
> Image previous is being displayed properly

Loads fine here (tested with local adapter only)

**Comment by brianteeman** [comment]
Not seen any problem here either

**Comment by joomdonation** [comment]
![banner_image](https://user-images.githubusercontent.com/977664/126060215-b14d850a-10c8-4bd9-bf62-60727eb44087.gif)

Hope it is a problem on my computer only. Here is what happens when I edit a banner which has image selected.

**Comment by chmst** [comment]
I see the "no image selectedt" message for a moment, then the banner. The image is ok. FF on win10.



![banner](https://user-images.githubusercontent.com/1035262/126061776-f83ed229-def4-4ebe-bdaf-d734dc499cff.gif)

**Comment by dgrammatiko** [comment]
Ok, observing the page source and the DOM after the JS was executed I have
<img width="1201" alt="Screenshot 2021-07-18 at 12 21 32" src="https://user-images.githubusercontent.com/3889375/126063721-ea42cc18-15f5-4e35-9a08-bf566e6e1463.png">

<img width="1100" alt="Screenshot 2021-07-18 at 12 21 48" src="https://user-images.githubusercontent.com/3889375/126063726-c87e2368-e62f-45ce-a27d-cec92da4c094.png">

Basically it's down to these lines https://github.com/joomla/joomla-cms/blob/3071cd68a5d17e9616144e5ee003f143122c2aa6/layouts/joomla/form/field/media.php#L77-L81

We either have to pass the image URL from the HTML helper or remove the hash part manually before checking with the `file_exists()`

**Comment by joomdonation** [comment]
I made PR https://github.com/joomla/joomla-cms/pull/34833 to fix the issue. Please help testing. Thanks !
